Makefile.am

```diff
@@ -1,5 +1,5 @@
 AUTOMAKE_OPTIONS = foreign
-SUBDIRS = src man lib/gitsh vendor spec
+SUBDIRS = src man vendor spec ext/gitsh lib/gitsh
```

Comment:
Does this order matter? Or: was this a technical change that I should learn about?